ts/screens/authentication/LandingScreen.tsx

```diff
@@ -89,15 +98,28 @@ const LandingScreen: React.SFC<Props> = props => {
       </Content>
 
       <View footer={true}>
+        {isCIEAvailable && (
+          <Button
+            block={true}
+            primary={true}
+            iconLeft={true}
+            onPress={undefined} // here navigate to identity card check
```

Comment:
```suggestion
            onPress={undefined} // TODO: here navigate to identity card check: https://www.pivotaltracker.com/story/show/169730093
```

Reply:
It seems you don't add the link to the pivotal story. Can you check it?

Reply:
Fixed